cmd/image-processing/main_test.go

```diff
@@ -193,177 +166,132 @@ var _ = Describe("Image Processing Resource", func() {
 
 		It("should fail in case --image does not exist", func() {
 			Expect(run(
-				"--image",
-				"docker.io/feqlQoDIHc/bcfHFHHXYF",
+				"--image", "docker.io/feqlQoDIHc/bcfHFHHXYF",
```

Comment:
For readability, I organized the flag and its value each in one line, similar to how you would do it in a shell script.